### ISSUE TITLE
Crash when using data binding

### ISSUE THREAD

**Issue by OneManStudioDotSe** [Omega-R/OmegaRecyclerView#140]
First of all, thank your for this library and your work. Really appreciated.

I have a problem and I could use some help.

I have the following layout:

```
<?xml version="1.0" encoding="utf-8"?>
<layout xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    xmlns:tools="http://schemas.android.com/tools">

    <androidx.coordinatorlayout.widget.CoordinatorLayout
        android:layout_width="match_parent"
        android:layout_height="match_parent"
        android:background="@color/background_whitish">

        <include
            android:id="@+id/appbar_holder"
            layout="@layout/new_element_appbar_collapsing_classic" />

        <RelativeLayout
            android:layout_width="match_parent"
            android:layout_height="match_parent"
            app:layout_behavior="@string/appbar_scrolling_view_behavior">

            <com.omega_r.libs.omegarecyclerview.OmegaRecyclerView
                android:id="@+id/discography_tracklist"
                android:layout_width="match_parent"
                android:layout_height="match_parent"
                android:scrollbars="vertical"
                tools:listitem="@layout/new_item_discography_track">

                <include
                    android:id="@+id/tracklist_header"
                    app:layout_section="header"
                    layout="@layout/element_tracklist_header" />

                <TextView
                    android:layout_width="match_parent"
                    android:layout_height="100dp"
                    android:text="Footer"
                    app:layout_section="footer"/>

            </com.omega_r.libs.omegarecyclerview.OmegaRecyclerView>

            <include
                android:id="@+id/loading"
                layout="@layout/new_element_loading"
                android:layout_width="match_parent"
                android:layout_height="wrap_content"
                android:layout_margin="@dimen/space_large"
                android:visibility="gone"/>

            <include
                android:id="@+id/error"
                layout="@layout/new_element_error"
                android:visibility="gone" />

            <include
                android:id="@+id/empty"
                layout="@layout/new_element_empty_content"
                android:visibility="gone" />

        </RelativeLayout>

    </androidx.coordinatorlayout.widget.CoordinatorLayout>

</layout>

```

which contains your recyclerView, a header and a footer. This layout, as well as the header layout are using data binding.

At the code, I access them like that:

 ```
override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        binding = DataBindingUtil.inflate(inflater, R.layout.new_fragment_album, container, false)

        return binding.root
    }

...

 binding.tracklistHeader.albumDescription.text = album!!.infoEn

        binding.discographyTracklist.hasFixedSize()
        binding.discographyTracklist.addItemDecoration(DividerItemDecoration(context, LinearLayoutManager(context).orientation))
        binding.discographyTracklist.adapter = tracksAdapter
        binding.discographyTracklist.setHeadersVisibility(true)
        binding.discographyTracklist.setFootersVisibility(true)
```

Unfortunately, it crashes with the following message:

```
java.lang.NullPointerException: Attempt to invoke virtual method 'void se.onemanstudio.rammstein.databinding.ElementTracklistHeaderBinding.invalidateAll()' on a null object reference
        at se.onemanstudio.rammstein.databinding.NewFragmentAlbumBindingImpl.invalidateAll(NewFragmentAlbumBindingImpl.java:72)
        at se.onemanstudio.rammstein.databinding.NewFragmentAlbumBindingImpl.<init>(NewFragmentAlbumBindingImpl.java:63)
        at se.onemanstudio.rammstein.databinding.NewFragmentAlbumBindingImpl.<init>(NewFragmentAlbumBindingImpl.java:44)
        at se.onemanstudio.rammstein.DataBinderMapperImpl.getDataBinder(DataBinderMapperImpl.java:314)
        at androidx.databinding.MergedDataBinderMapper.getDataBinder(MergedDataBinderMapper.java:74)
        at androidx.databinding.DataBindingUtil.bind(DataBindingUtil.java:199)
        at androidx.databinding.DataBindingUtil.inflate(DataBindingUtil.java:130)
        at androidx.databinding.DataBindingUtil.inflate(DataBindingUtil.java:95)
        at se.onemanstudio.rammstein.v3.ui.biosandalbums.discography.AlbumFragment.onCreateView(AlbumFragment.kt:38)
        at androidx.fragment.app.Fragment.performCreateView(Fragment.java:2600)
```

Does the library support data binding? What else might be the reason for the crash?

**Comment by scrinoman** [comment]
Hi, @OneManStudioDotSe.
The problem was in our Sections feature, headers and footers were not added to RecyclerView as childs at the time Data Binding requests views.

We fixed it in release 1.9.8.

Comment back, if you problem wasn't resolved.

**Comment by OneManStudioDotSe** [comment]
It works smoothly now. Thanks for the quick reply and fix :)
